frontend/src/routes/(authenticated)/admin/[[tab=adminTab]]/AdminProjects.svelte

```diff
@@ -53,24 +55,31 @@
 
 <ConfirmDeleteModal bind:this={deleteProjectModal} i18nScope="delete_project_modal" />
 <div>
-  <div class="flex justify-between items-center">
-    <h2 class="text-2xl flex gap-4 items-end">
-      {$t('admin_dashboard.project_table_title')}
-      <Badge>
-        <span class="inline-flex gap-2">
-          {$number(shownProjects.length)}
-          <span>/</span>
-          {$number(filteredProjects.length)}
+  <AdminTabs activeTab="projects">
+    <div class="flex gap-4 justify-between grow">
+      <div class="flex gap-4 items-center">
+        {$t('admin_dashboard.project_table_title')}
+        <div class="contents max-xs:hidden">
+          <Badge>
+            <span class="inline-flex gap-2">
+              {$number(shownProjects.length)}
+              <span>/</span>
+              {$number(filteredProjects.length)}
+            </span>
+          </Badge>
+        </div>
+      </div>
+      <!-- <a> totally breaks SSR 🙃 -->
+      <svelte:element this={browser ? 'a' : 'div'}
```

Comment:
huh? is this the svelte kit bug you mentioned?

Reply:
Yup. If this is an `<a>` during SSR, the HTML is an absolute mess.

Reply:
FYI I'm now using a button for tabs, so I could change this back to a normal `<a>`
But, I discovered that we have the same problem in our member-badges. I seems to results from nesting either `<button>`s or `<a>`s.
I've now commented about this on a [similar issue](https://github.com/sveltejs/kit/issues/8345#issuecomment-1976802414).

Seeing as you're not supposed to nest buttons or a's, I guess it's our own fault 😆. Fixing the member-badge is easy. I'll push that to develop.